src/archivist/filter/index.js

```diff
@@ -20,6 +20,18 @@ const { CiceroMarkTransformer } = ciceroMark;
 
 const ciceroMarkTransformer = new CiceroMarkTransformer();
 
+/**
+ * Filter content and convert it to markdown
+ *
+ * @param {Object} config - Filter configuration
+ * @param {string} config.content - Content to filter
+ * @param {string} config.mimeType - Mime type of the given content
+ * @param {Object} config.documentDeclaration - Document declaration used to describe document in Open Terms Archive
```

Comment:
Not clear. Provide a reference to a class or to further documentation, or just remove this altogether since you document the members of this object 🙂

Reply:
I removed as DocumentDeclaration is a class that is not exposed yet and thus is not really useable when imported as a module.